diff -urN php-pear-File_SMBPasswd-1.0.1.orig/File_SMBPasswd-1.0.1/SMBPasswd.php php-pear-File_SMBPasswd-1.0.1/File_SMBPasswd-1.0.1/SMBPasswd.php
--- php-pear-File_SMBPasswd-1.0.1.orig/File_SMBPasswd-1.0.1/SMBPasswd.php	2004-09-14 18:08:45.000000000 +0200
+++ php-pear-File_SMBPasswd-1.0.1/File_SMBPasswd-1.0.1/SMBPasswd.php	2004-12-22 10:37:17.804807736 +0100
@@ -180,7 +180,7 @@
      * @return mixed returns PEAR_Error, if the user already exists
      * @access public
      */
-    function addAccountEncrypted($user, $userid, $lmhash = '', $nthash = '', $comment = '', $flags = '[U           ]') 
+    function addAccountEncrypted($user, $userid, $lmhash = '', $nthash = '', $comment = '', $flags = '[U          ]') 
     {
         if (empty($lmhash)) $lmhash = str_repeat('X', 32);
         if (empty($nthash)) $nthash = str_repeat('X', 32);
@@ -213,7 +213,7 @@
      * @return mixed returns PEAR_Error, if the user already exists
      * @access public
      */
-    function addAccount($user, $userid, $pass, $comment = '', $flags = '[U           ]') 
+    function addAccount($user, $userid, $pass, $comment = '', $flags = '[U          ]') 
     {
         if (empty($pass)) {
             return $this->addAccountEncrypted($user, $userid, '', '', $comment, $flags) ;
@@ -242,7 +242,7 @@
      */
     function addUser($user, $userid, $pass, $comment = '')
     {
-        return $this->addAccount($user, $userid, $pass, $comment, '[U           ]');
+        return $this->addAccount($user, $userid, $pass, $comment, '[U          ]');
     }
 
     /**
@@ -257,7 +257,7 @@
      */
     function addMachine($machine, $userid, $comment = '')
     {
-        return $this->addAccount($machine . '$', $userid, $machine, $comment, '[W           ]');
+        return $this->addAccount($machine . '$', $userid, $machine, $comment, '[W          ]');
     }
     
     /**
